packages/tyxml-lwd/tyxml-lwd.0.2/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Make reactive webpages in Js_of_ocaml using Tyxml and Lwd"
+maintainer: ["fred@tarides.com"]
+authors: ["Frédéric Bour"]
+license: "MIT"
+homepage: "https://github.com/let-def/lwd"
+doc: "https://let-def.github.io/lwd/doc"
+bug-reports: "https://github.com/let-def/lwd/issues"
+depends: [
+  "dune" {>= "2.7" & >= "2.7"}
```

Comment:
```suggestion
  "dune" {>= "2.7"}
```
To fix this, you should remove `dune` from the dependencies in the dune-project file. It is added automatically